sdk/eventhub/event-hubs/test/README.md

```diff
@@ -2,15 +2,15 @@
 
 To test this project, make sure to build it by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building), then follow the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
 
-The Azure Event Hubs client does not have any recorded tests and so, all the tests require an Azure Event Hubs namespace to be set up beforehand with at lease a single Event Hub instance in it. You can use existing Azure resources for the live tests, or generate new ones by using our [New-TestResources.ps1](https://github.com/Azure/azure-sdk-for-js/blob/master/eng/common/TestResources/New-TestResources.ps1) script, which will use an [ARM template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/test-resources.json) that already has all of the the necessary configurations.
+## Azure resources 
+
+The Azure Event Hubs client does not have any recorded tests and so, all the tests require an Azure Event Hubs namespace to be set up beforehand with at least a single Event Hub instance in it. You can use existing Azure resources for the live tests, or generate new ones by using our [New-TestResources.ps1](https://github.com/Azure/azure-sdk-for-js/blob/master/eng/common/TestResources/New-TestResources.ps1) script, which will use an [ARM template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/test-resources.json) that already has all of the the necessary configurations.
 
 The Azure resources that are used by the tests in this project are:
 
-- An [Azure Event Hubs namespace](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-features#namespace). Your Azure application needs to be assigned as the **owner** of this Azure Key Vault. The steps are provided [below](#AAD-based-authentication).
-- Some very basic Network configurations for the Event Hubs namespace.
-- A [consumer group](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-features#consumer-groups) to publish and subscribe to the Event Hubs namespace created.
+- An [Azure Event Hubs namespace](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-features#namespace) with atleast one Event Hub instance in it.
 - An [Azure Storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) configured to provide [blob storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#blob-storage-resources).
-- An [Azure IoT Hub account](https://docs.microsoft.com/en-us/azure/iot-hub/about-iot-hub).
+- An Azure Active Directory application for the tests to use. See the [AAD based authentication](#AAD-based-authentication) for steps to register the application.
 
 To run the live tests, you will also need to set the below environment variables:
 
```

Comment:
Just saying...
In the line below, `- TEST_MODE: Should have "live" assigned.` is not required.

Reply:
lol, thats right!

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/8602/commits/b57c4cf263a50d4315fda5ef6f8f13bde9c7c519